example/Vagrantfile

```diff
@@ -1,11 +1,17 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+unless Vagrant.has_plugin?("vagrant-docker-compose")
+        system('vagrant plugin install vagrant-docker-compose')
+        raise("Plugin installed. Run command again.");
+end
```

Comment:
Could you make a couple changes to this part:

1. Bring these indentations in at only two spaces for consistency.
2. Replace `system('...')` with `system("...")` for consistency.
3. Replace the `raise` with `puts` and `exit`, so that the user sees only the message they need to see, and don't need to read the six line vagrant error message to find our error message within.

Suggest:
```ruby
unless Vagrant.has_plugin?("vagrant-docker-compose")
  system("vagrant plugin install vagrant-docker-compose")
  puts "Dependencies installed, please try the command again."
  exit
end
```